changelog/debug-nothrow.dd

```diff
@@ -0,0 +1,14 @@
+Nothrow code can now be used in debug blocks
```

Comment:
> Code in `debug` blocks will now ignore `nothrow`

Or something related. `nothrow` code can already be used in `debug` blocks :)

Reply:
Fixed :O